tools/parser_sql/lexer/LexerSplitter.php

```diff
@@ -53,7 +53,7 @@ class LexerSplitter {
 
     protected static $splitters = array('<=>', "\r\n", '!=', '>=', '<=', '<>', '<<', '>>', ':=', '\\', '&&', '||', ':=',
                                        '/*', '*/', '--', '>', '<', '|', '=', '^', '(', ')', "\t", "\n", "'", '"', '`',
-                                       ',', '@', ' ', '+', '-', '*', '/', ';');
+                                       ',', '@', ' ', '+', '-', '*', '/', ';');,
```

Comment:
not sure about this one..

Reply:
yes, it looks weird. I will revert it.

Reply:
Should be: `',', '@', ' ', '+', '-', '*', '/', ';',);`

Reply:
and `);` on next line

Reply:
Fixed